### PR TITLE
move debugging tips to separate optional lab

### PR DESCRIPTION
**Pull request by MHBauer**
this is a grab bag of topics that could possibly included in another lab at point of use, but I feel is better isolated to a separate location for reference.

### CODE REVIEW

**Comment by duglin** [comment]
Seems like we should at least mention this lab in the readme.

**Comment by MHBauer** [comment]
@duglin I agree, we need entries for 3 and 4 as well.

**Comment by MHBauer** [comment]
Down here, right https://github.com/IBM/kube101/blame/master/workshop/README.md#L161-L167 ?

**Comment by MHBauer** [comment]
@brahmaroutu to handle

**Comment by MHBauer** [comment]
@duglin I wrote up the direct link to the lab text. Should be done with this PR for now.

File: workshop/README.md [hunk 1]
```diff
@@ -165,3 +165,9 @@ IBM Cloud provides the capability to run applications in containers on Kubernete
 [Lab 1](Lab1): This lab walks through creating and deploying a simple "guestbook" app written in Go as a net/http Server and accessing it.
 
 [Lab 2](Lab2): Builds on lab 1 to expand to a more resilient setup which can survive having containers fail and recover. Lab 2 will also walk through basic services you need to get started with Kubernetes and the IBM Cloud Container Service
+
+[Lab 3](Lab3): Builds on lab 2 by increasing the capabilities of the deployed Guestbook application. This lab covers basic distributed application design and how kubernetes helps you use standard design practices.
+
+[Lab 4](Lab4): How to enable your application so Kubernetes can automatically monitor and recover your applications with no user intervention.
+
+[Lab D](LabD): Debugging tips and tricks to help you along your Kubernetes journey. This lab is useful reference that does not follow in a specefic sequence of the other labs.
```
Comment (duglin) [inline workshop/README.md]: s/specefic/specific/

Comment (MHBauer) [reply workshop/README.md]: @duglin you can edit it directly in the browser. I gave you permission on my fork.

**Pull request merged**